test/expected.txt

```diff
@@ -212,11 +230,12 @@
 || 
 || 0 persons listed!
 || ===================================================
-|| Enter command: || [Command entered:  find betsy]
+|| Enter command: || [Command entered:  find Betsy]
+|| 	1. Betsy Choo Tags: [secretive]
 || 
-|| 0 persons listed!
+|| 1 persons listed!
 || ===================================================
-|| Enter command: || [Command entered:  find Betsy]
+|| Enter command: || [Command entered:  find betsy]
 || 	1. Betsy Choo Tags: [secretive]
```

Comment:
This works, but I am not sure why you decided to swap `find betsy` with `find Betsy` commands...